server.js

```diff
@@ -1,50 +1,68 @@
 /**
  * Created by anatal on 3/17/17.
  */
-var http = require('http');
+const http = require('http');
 // Serve client side statically
-var express = require('express');
-var app = express();
-var server = http.createServer(app);
-var bodyParser = require('body-parser');
-var fs = require('fs');
+const express = require('express');
+const app = express();
+const server = http.createServer(app);
+const bodyParser = require('body-parser');
+const fs = require('fs');
+const spawnSync = require('child_process').spawnSync;
 
 app.use(function(req, res, next) {
-    res.header("Access-Control-Allow-Origin", "*");
-    res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
-    next();
+  res.header('Access-Control-Allow-Origin', '*');
+  res.header(
+    'Access-Control-Allow-Headers',
+    'Origin, X-Requested-With, Content-Type, Accept'
+  );
+  next();
 });
 
-var rawBodySaver = function (req, res, buf, encoding) {
-    if (buf && buf.length) {
-        req.rawBody = buf;
-        console.log(buf.length, ' bytes written');
+const rawBodySaver = function(req, res, buf) {
+  if (buf && buf.length) {
+    req.rawBody = buf;
+    console.log(buf.length, ' bytes written');
 
-        // first we save the file as opus
-        var id = Math.random() * Date.now() ;
-        fs.writeFileSync(id + '.opus' , req.rawBody, (err) => {
-            if (err) throw err;
-        });
+    // first we save the file as opus
+    const id = Math.random() * Date.now();
+    fs.writeFileSync(id + '.opus', req.rawBody, err => {
+      if (err) throw err;
+    });
 
-        // then we convert it from opus to raw pcm
-        var resultcurl = require('child_process').spawnSync('opusdec', ['--rate', '16000', id + '.opus', id + '.raw']);
+    // then we convert it from opus to raw pcm
+    let resultcurl = spawnSync('opusdec', [
+      '--rate',
+      '16000',
+      id + '.opus',
+      id + '.raw'
+    ]);
 
-        // send to the asr server
-        var resultcurl = require('child_process').spawnSync('curl', ['-H',"Content-Type: application/octet-stream","--data-binary", "@" + id + '.raw',"http://10.252.24.90/asr?endofspeech=false&nbest=10"] );
```

Comment:
I notice we were defining `resultcurl` twice (here and on line 30 above), and both times doing a `require('child_process')`, so I moved the `require()` statements up top with the others.

**prettier** auto formatted the long strings onto separate lines below for better readability.